tests/Moq.Analyzers.Test/Moq.Analyzers.Test.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)/src/Moq.Analyzers/Moq.Analyzers.csproj" AddPackageAsOutput="true" />
+    <ProjectReference Include="..\..\src\Moq.CodeFixes\Moq.CodeFixes.csproj" />
```

Comment:
I don't think you should need to do this? Shouldn't it be brought in via the reference in analyzers?